### PR TITLE
[PHI]Unify fluid kernel (Part5)

### PR DESCRIPTION
**Pull request by YuanRisheng**
### PR types

Others
### PR changes

Others
### Describe

Pcard-67000
将以下Fluid op注册到phi中：
      sample_logits
      sampling_id
      seed
      send_and_recv
      send_v2
      sequence_concat

### CODE REVIEW

**Review by jiweibo — APPROVED** [review]

**Review by huangjiyi — APPROVED** [review]
LGTM

**Review by zyfncg — APPROVED** [review]

**Review by heavyrain-lzy — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by wanghuancoder — APPROVED** [review]
